automatic/git.install/git.install.nuspec

```diff
@@ -29,7 +29,7 @@ Git for Windows focuses on offering a lightweight, native set of tools that brin
 
 - The package uses default install options minus cheetah integration and desktop icons. Cheetah prevents a good upgrade scenario, so it has been removed.
 - For disabling the Git Credential Manager see the [Disable Git Credential Manager package](https://chocolatey.org/packages/git-disable-gcm).
```

Comment:
We could add here that either the ` NoCredentialManager` flag or the `Disable Git Credential Manager package` could be used.